README.md

```diff
@@ -195,8 +198,8 @@ Imports an **Array** of **Daru::DataFrame**s from a **.html** file or website.
     require 'daru/io/importers/html'
 
     #! Usage from Daru::IO
-    df1 = Daru::IO::Importers::HTML.new('https://some/url/with/tables', match: 'market', name: 'Shares analysis').call
-    df2 = Daru::IO::Importers::HTML.new('path/to/file.html', match: 'market', name: 'Shares analysis').call
+    df1 = Daru::IO::Importers::HTML.new(match: 'market', name: 'Shares analysis').read('https://some/url/with/tables')
```

Comment:
What I've said above relates to all importers, but here it is extremely visible:
```ruby
importer = Daru::IO::Importers::HTML.new('https://some/url/with/20Mb/of/tables') # URL read once, Nokogiri document parsed once
df1 = importer.call(match: 'prices')
df2 = importer.call(match: 'goods')
df3 = importer.call(match: 'clients')
```
Do you feel it?..

Reply:
Ah, I can see this. But I feel that the existing setup also makes sense when gathering similar data from multiple sources. Something like importing from GitHub API of various repositories,

```ruby
json_importer = Daru::IO::Importers::JSON.new(RepositoryName: '$..full_name', Stars: '$..stargazers_count', Size: '$..size', Forks: '$..forks_count')

%w[athityakumar zverok v0dro].map do |username|
    json_importer.read("https://api.github.com/users/#{username}/repos")
end.reduce(:concat)
```

But of course, your suggestion does better when the data source is pretty huge. We definitely don't want to read again & again for sure. I can change that, and maybe it can be used like this even for the example I gave like - 

```ruby
json_options = {RepositoryName: '$..full_name', Stars: '$..stargazers_count', Size: '$..size', Forks: '$..forks_count'}

%w[athityakumar zverok v0dro].map do |username|
    Daru::IO::Importers::JSON.read("https://api.github.com/users/#{username}/repos").call(json_options)
end.reduce(:concat)
```

No issues, refactoring this shouldn't take much time (hopefully). 😅

Reply:
Yep, you last code example is "right" way (as for me). Creating importer just by JSONPath's provide no value (as nothing "heavy" is done in create, and reparsing JSONPath on each call is cheap)

Reply:
@zverok - Acknowledged. I'll update this PR by tomorrow. 👍

Reply:
(That's why it is sometimes better to discuss code, than just ideas in GitHub issues... We could do faster if it would not be so hard for me to imagine everything beforehands. Sorry for that.)

Reply:
@zverok - Added the changes. Please review again. 😄